fe/fe-core/src/main/java/org/apache/doris/nereids/trees/expressions/ExprId.java

```diff
@@ -17,13 +17,15 @@
 
 package org.apache.doris.nereids.trees.expressions;
 
+import org.apache.doris.common.Id;
+
 import java.util.Objects;
 import java.util.UUID;
 
 /**
  * UUID for Expression in Nereids.
  */
-public class ExprId {
+public class ExprId extends Id<ExprId> {
     private final long id;
     private final UUID jvmId;
```

Comment:
remove these lines instead with parent class's attribute: id

Reply:
done

Reply:
And removed the jvmId by the way